src/tools/fuzzing/fuzzing.cpp

```diff
@@ -820,6 +864,35 @@ Expression* TranslateToFuzzReader::makeImportTableSet(Type type) {
     Type::none);
 }
 
+Expression* TranslateToFuzzReader::makeImportCallExport(Type type) {
+  // The none-returning variant just does the call. The i32-returning one
+  // catches any errors and returns 1 when it saw an error. Based on the
+  // variant, pick which to call, and the maximum index to call.
+  Name target;
+  Index maxIndex = wasm.exports.size();
+  if (type == Type::none) {
+    target = callExportImportName;
+  } else if (type == Type::i32) {
+    target = callExportCatchImportName;
+    // This never traps, so we can be less careful, but we do still want to
+    // avoid trapping a lot as executing code is more interesting.
+    maxIndex = (maxIndex + 1) * 2;
```

Comment:
Maybe multiply by less than two here? I don't think we need half of the calls to be out of bounds.

Reply:
I was thinking (but I see I didn't document :smile: ) that this is fine because we are doing this as we go, that is, we are still adding functions (we don't know how many in advance). So even larger offsets will have a good chance to be in bounds. What do you think?

Reply:
aha, that seems reasonable, I suppose.